source/_integrations/rfxtrx.markdown

```diff
@@ -344,11 +344,22 @@ action:
     event: 0b1111e003af16aa10000060
 ```
 
+Alternatively:
+
+[![Open your Home Assistant instance and show your service developer tools.](https://my.home-assistant.io/badges/developer_services.svg)](https://my.home-assistant.io/redirect/developer_services/)
+- Select: `RFXCOM RFXtrx: Send` from the Service drop-down menu.
+
+```yaml
+service: rfxtrx.send
+data:
+  event: "0b1111e003af16aa10000060"
+```
+
 ## Generate codes
 
 If you need to generate codes for switches and lights, you can use a template (useful, for example, COCO switches).
 
-- Go to home-assistant-IP:8123/dev-template
+[![Open your Home Assistant instance and show your template developer tools.](https://my.home-assistant.io/badges/developer_template.svg)](https://my.home-assistant.io/redirect/developer_template/)
```

Comment:
Please don't use the badges, just link inline text.

Additionally, use the `{% my .. %}` plugin used in the documentation instead of hard Markdown/static links.